README.en.md

```diff
@@ -494,3 +500,43 @@ as possible, we would need to know information like the followings.
 | Log                  | Error log when an error occurs                                        |
 | Request restriction  | Request to `wovn.global.ssl.fastly.net` with 443 port must be allowed |
 | Using SSI            | Whether you are using SSI(Server Side Includes)                       |
+
+### Wovn Diagnostics Tool
+
+WOVN.php ships with a diagnostics tool that automatically gathers information for debugging purposes. This tool is shippped disabled by default.
+
+This diagnostics tool is meant for clients running WOVN.php with Apache. To enable the Wovn Diagnostics Tool, please do the following tasks:
+
+1. Uncomment the relevant lines in the `.htaccess` files in your documents root *and* the WOVN.php folder.
+
+2. Create a `.htpasswd` file by running the command `htpasswd -cB .htpasswd wovn_diagnostics` inside the WOVN.php folder. This will create a password for username `wovn_diagnostics`.
+
+3. Create an empty `wovnphp_diagnostics.html` file inside the WOVN.php folder.
+
+4. Access the diagnostic tool at `yourwebsite.com/WOVN.php/diagnostics.php`. You will need to login using the password created in step 2.
+
+5. Follow the on screen instructions to get the diagnostic report, which will be written to the `wovnphp_diagnostics.html`.
+
+Please only enable the diagnostics tool when it is necessary to do so.
+
+#### If you are upgrading
+If you are upgrading from an older version of WOVN.php, please manually add the following lines to your `.htaccess` files:
+
+**Inside your documents root**
+```
+<IfModule mod_rewrite.c>
+  RewriteEngine On
+
+  # Intercept only static content: html and htm urls
+  # Warning: do not remove this line or other content could be loaded
+  RewriteCond %{REQUEST_URI} /$ [OR]
+  RewriteCond %{REQUEST_URI} \.(html|htm|shtml|php|php3|phtml)
+  RewriteCond %{QUERY_STRING} !^off_wovn_php=1$
+  # Use the wovn_index.php to handle static pages
+  RewriteRule .? wovn_index.php [L]
```

Comment:
This section of the `.htaccess` is meant only for static websites. I guess you are adding it for `RewriteCond %{QUERY_STRING} !^off_wovn_php=1$` but it will have unwanted impact on PHP sites.

_Some history: Because it is difficult to fully support any website configuration with `.htaccess`, we deprecated wovnphp and created WOVN.php with two modes: a PHP mode where our script must be required from the PHP code that limits bugs related to content discovery (because we rely on devs rather than our `.htaccess`), and a static mode for non-PHP websites that relies on `.htaccess`_.